src/chromatix/functional/propagation.py

```diff
@@ -63,7 +64,7 @@ def transfer_propagate(
     n: float,
     N_pad: int,
     cval: float = 0,
-    kykx: Array = jnp.zeros((2,)),
+    kykx: Array = (0.0, 0.0),
```

Comment:
Same as before, we need to change the type, and not sure we can parallelize over this as no longer an Array?